### PR TITLE
Local 16bpp fixes

### PR DESCRIPTION
**Pull request by akallabeth**
Fixes issues mentioned in #3709

### CODE REVIEW

File: libfreerdp/codec/planar.c [hunk 1]
```diff
@@ -223,12 +223,57 @@ static INT32 planar_decompress_plane_rle(const BYTE* pSrcData, UINT32 SrcSize,
...
+
+			return TRUE;
+
+		case PIXEL_FORMAT_BGRX32:
+			for (x = 0; x < width; x++)
+			{
+				*(*ppRgba)++ = *(*ppB)++;
+				*(*ppRgba)++ = *(*ppG)++;
+				*(*ppRgba)++ = *(*ppR)++;
+				*(*ppRgba)++ = *(*ppA)++;
```
Comment (mfleisz) [inline libfreerdp/codec/planar.c]: shouldn't this be ```*(*ppRgba)++ = 0xFF```?

File: libfreerdp/codec/planar.c [hunk 2]
```diff
@@ -223,12 +223,57 @@ static INT32 planar_decompress_plane_rle(const BYTE* pSrcData, UINT32 SrcSize,
...
+			}
+
+			return TRUE;
+
+		default:
+			for (x = 0; x < width; x++)
+			{
+				BYTE alpha = (ppA) ? *(*ppA)++ : 0xFF;
```
Comment (mfleisz) [inline libfreerdp/codec/planar.c]: Could improve performance a bit by making two for-loops - one for alpha and one for non-alpha formats

**Comment by bmiklautz** [comment]
@akallabeth nice! Definitely fixes the 16 color problems reported.

**Comment by akallabeth** [comment]
@mfleisz Could you review my updated pull?
@bmiklautz Ok, rebased to current master and removed merge conflicts.

**Pull request merged**